gigadb/app/tools/files-metadata-console/tests/functional/CheckValidURLsCest.php

```diff
@@ -16,6 +16,9 @@ class CheckValidURLsCest
         "https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100006/phylogeny_study_update/",
     ];
 
+    /**
+     * @skip Skipping this test due to unreliable resolution of URLs
+     */
```

Comment:
This test is fixed in PR https://github.com/gigascience/gigadb-website/pull/1745/files#diff-fe4bd02276e47a91f99dc3db159df2b55286b8e6c910101bcf23700e86a6f643

Reply:
I removed skip tag now.